tools/devops/automation/templates/tests/stage.yml

```
@@ -169,3 +176,34 @@ stages:
         statusContext: ${{ parameters.statusContext }}
         vsdropsPrefix: ${{ parameters.vsdropsPrefix }}
         testPrefix: ${{ parameters.testPrefix }}
+
+  - job: publish_skipped_tests
+    ${{ if eq(parameters.parseLabels, true) }}:
+      condition: ne(dependencies.configure.outputs['labels.build-package'], '')
+    ${{ else }}:
+      condition: false
+    displayName: 'GitHub comment - Skipped tests'
+    timeoutInMinutes: 1000
+    dependsOn: # has to wait for the tests to be done AND the data to be uploaded
+    - ${{ if eq(parameters.parseLabels, true) }}:
+      - configure
```

Is the condition here necessary?

```suggestion
    -  configure
```

---

Does make no much diff, less lines is better anyway.